### PR TITLE
Setup the repo for expeditor to manage the changelog/promotion

### PR DESCRIPTION
**Pull request by marcparadise**
Add this  repository to our build/CI process. 

A separate PR will be submitted to add buildkite.

### CODE REVIEW

File: .expeditor/config.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+promote:
+  action:
+    - built_in:publish_rubygems
+
+subscriptions:
+  - workload: ruby_gem_published:train-core-*
+    actions:
+      - bash:.expeditor/update_dep.sh
+  # TODO - change to chef 15 once 15 is released.
+  - workload: ruby_gem_published:chef-14*
```
Comment (tas50) [inline .expeditor/config.yml]: You might want to just make this 15 now. There won't be another 14 release before 15.

File: .expeditor/config.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  - built_in:bump_version:
+      ignore_labels:
+        - "Version: Skip Bump"
+        - "Expeditor: Skip Version Bump"
+        - "Expeditor: Skip All"
+  - bash:.expeditor/update_version.sh:
+      only_if: built_in:bump_version
+  - built_in:update_changelog:
+      ignore_labels:
+        - "Meta: Exclude From Changelog"
```
Comment (tas50) [inline .expeditor/config.yml]: The meta label can go away. This doesn't exist

File: .expeditor/config.yml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+rubygems:
+  - chef-core
+  - chef-core-actions
+
+# These actions are taken, in order they are specified, anytime a Pull Request is merged.
+merge_actions:
+  - built_in:bump_version:
+      ignore_labels:
+        - "Version: Skip Bump"
```
Comment (tas50) [inline .expeditor/config.yml]: The version label doesn't exist and can go away

File: .expeditor/config.yml [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+      ignore_labels:
+        - "Meta: Exclude From Changelog"
+        - "Expeditor: Exclude From Changelog"
+        - "Expeditor: Skip All"
+  - built_in:build_gem:
+      only_if:
+        - built_in:bump_version
+
+promote:
+  action:
```
Comment (tas50) [inline .expeditor/config.yml]: I believe this should be actions now. I'd look at the chef config. This looks like a mildly dated expeditor config.

File: .expeditor/config.yml (hunk 1 above)
Comment (marcparadise) [reply .expeditor/config.yml]: wfm

File: .expeditor/config.yml (hunk 4 above)
Comment (marcparadise) [reply .expeditor/config.yml]: That makes sense, I pulled it from `chef-apply` which got set up almost a year ago. I'll compare `chef` and update .

**Comment by marcparadise** [comment]
I'm getting the expeditor error about the channel not being valid, though the instructions for fixing it don't work. I've kept the notifications channel as chef-ws-notify for now, until we can resolve it.

**Pull request merged**